_posts/2016-04-05-ace-the-technical-interview.markdown

```diff
@@ -0,0 +1,40 @@
+author:       gabby_losch
+summary:      Technical interviews don't need to be soulless. Here's the perfect answer, verified by science.
+image:        //res.cloudinary.com/wework/image/upload/v1459867978/technical-interviews.jpg
+categories:   engineering
+---
+
+### Standard Interviews
+
+If you've been on either side of a technical interview, you've probably heard questions about sorting algorithms, discussed [Big O](https://en.wikipedia.org/wiki/Big_O_notation), asked or been asked how that function could be made [faster, better, harder, stronger](https://www.youtube.com/watch?v=GDpmVUEjagg), and you've also likely wanted to run out of the room [screaming](https://www.youtube.com/watch?v=H07zYvkNYL8). Or maybe you did run out of the room screaming. 
+
+Technical interviews are hard for both interviewer and interviewee. As the interviewer, you are tasked with determining the scope of knowledge possessed by the stranger sitting across the table. _Can he navigate common technical hangups? Is she well-versed in cross-browser compatibility issues? Will he lock up or --¡gasp!-- delete our entire production database by accident?_ On top of the technical knowhow, _will this person be a pleasant addition to the team, who actively and intelligently contributes to sprint-planning meetings and retros, and helps the team do its collective best work?_ Oh, and you have 45 minutes to figure all that out. Sure, no problem. 
+
+As the interviewee, you need to speak articulately (stifle those 'ummm's), answer elaborate technical scenarios or obscure computer science zingers with someone looking over your shoulder and without obviously stumbling, and present yourself as a real human adult person with a personality.
+
+### How to Nail it
+
+Preparing for every technical question is impossible. Every company has a different approach, a different tech stack, and will ask you something you haven't seen before. That's obviously no excuse to not prepare, and working through a bunch of questions on [Project Euler](https://projecteuler.net/archives) will help you stay sharp, but when the time comes, it's often not enough to rattle off a technically-correct solution. Adding a little flair will help you stand out from the crowd. 
+
+Here's a question we commonly ask during technical interviews: 
+> Describe what happens when a person types a URL into their browser and hits 'enter'
+
+This question is great for a number of reasons:
+
+- it's conversational, so the interviewee doesn't have to panic at the dreaded white board marker
+- the answer given can lead to follow-up questions and further discussion
+- the topic is massively practical -- anyone interviewing for an engineering role should understand how the modern internet works
+- it is language/tech stack-agnostic
+
+During my technical interviews for the WeWork engineering team, I was asked this question (twice, in fact!), and here's the answer I gave (kind of):
```

Comment:
I think we can leave out "(twice, in fact!)", we want to give the (perhaps wrong) impression that we're organized